src/dynamic-page.scss

```diff
@@ -398,12 +397,8 @@ $block: #{$fd-namespace}-dynamic-page;
       padding: 0;
     }
 
-    .#{$block}__title {
-      padding: 0.6875rem 0 0 0;
```

Comment:
Fiori3 spec has these values specified for `padding-top`. Is it okay to remove these values?

Reply:
Hi @kavya-b , these paddings are added to the title element to center it vertically in the container. We can achieve the centering with Flex. When only title is used, the padding top is causing problems with the vertical alignment.
See the example from another project:
<img width="1353" alt="Screen Shot 2021-03-09 at 10 51 41 AM" src="https://user-images.githubusercontent.com/39598672/110498620-7d0f4f00-80c5-11eb-8737-d5bdf35dcdcb.png">